src/main/java/duke/tasklist/TaskList.java

```diff
@@ -0,0 +1,155 @@
+package duke.tasklist;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+import duke.dukeexception.DukeException;
+import duke.dukeexception.DukeTaskNotFound;
+import duke.task.Task;
+
+/**
+ * A List that is used to contain the Tasks inputted by the user.
+ */
+public class TaskList {
+    private ArrayList<Task> list;
```

Comment:
Since there is a list interface already in Java, should this variable name be changed to prevent confusion?